scanner/base.go

```diff
@@ -621,6 +648,10 @@ func (l *base) scanLibraries() (err error) {
 
 	l.log.Info("Scanning Language-specific Packages...")
 
+	if err := trivyInitOnce(); err != nil {
+		return xerrors.Errorf("Failed to initialize Trivy. err: %w", err)
+	}
```

Comment:
refs. https://github.com/future-architect/vuls/pull/1806/files#r1494724483
```suggestion
	if err := trivyLoggerInitOnce(); err != nil {
		return xerrors.Errorf("Failed to init trivy logger. err: %w", err)
	}
```